lib/pure/terminal.nim

```diff
@@ -17,6 +17,38 @@
 ## ``showCursor`` before quitting.
 
 import macros
+import strformat
+from strutils import toLowerAscii
+import colors
+
+const
+  hasThreadSupport = compileOption("threads")
+
+when not hasThreadSupport:
+  import tables
+  var
+    colorsFGCache: Table[Color, string]
+    colorsBGCache: Table[Color, string]
+  when not defined(windows):
+    var
+      styleCache: Table[int, string]
+
+var
+  trueColorIsSupported {.threadvar.}: bool
```

Comment:
These threadvars are still not correct!

Reply:
I can't understand: why a threadvars in other modules are correct?